packages/phone-number-privacy/src/match-making/get-contact-matches.ts

```diff
@@ -14,9 +13,9 @@ export async function handleGetContactMatches(request: Request, response: Respon
       return
     }
     authenticateUser()
-    const matchedContacts: ContactMatch[] = (
+    const matchedContacts: any = (
```

Comment:
Should this be a string[]?

Reply:
I wanted to keep it as an object in case we ever wanted to return anything else.  I guess I could keep the ContactMatch object so I can strictly define the type of `phoneNumber` as string though